src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts

```diff
@@ -30,23 +30,35 @@ import {
 import { hasMinAuthenticatorOptions, loc, updateTransactionWithNextStep } from '../../../util';
 
 const getTitleText = (challengeMethod: string) => {
-  if (challengeMethod === CHALLENGE_METHOD.APP_LINK) {
-    return loc('appLink.title', 'login');
-  }
-  if (challengeMethod === CHALLENGE_METHOD.UNIVERSAL_LINK) {
-    return loc('universalLink.title', 'login');
+  switch (challengeMethod) {
+    case CHALLENGE_METHOD.APP_LINK:
+      return loc('appLink.title', 'login');
+
+    case CHALLENGE_METHOD.CHROME_DTC:
+      return loc('chrome_dtc.title', 'login');
+
+    case CHALLENGE_METHOD.UNIVERSAL_LINK:
+      return loc('universalLink.title', 'login');
+
+    default:
+      return loc('customUri.title', 'login');
   }
-  return loc('customUri.title', 'login');
 };
 
 const getDescriptionText = (challengeMethod: string) => {
-  if (challengeMethod === CHALLENGE_METHOD.APP_LINK) {
-    return loc('appLink.content', 'login');
-  }
-  if (challengeMethod === CHALLENGE_METHOD.UNIVERSAL_LINK) {
-    return loc('universalLink.content', 'login');
+  switch (challengeMethod) {
+    case CHALLENGE_METHOD.APP_LINK:
+      return loc('appLink.content', 'login');
+
+    case CHALLENGE_METHOD.UNIVERSAL_LINK:
+      return loc('universalLink.content', 'login');
+
+    case CHALLENGE_METHOD.CHROME_DTC:
+      return '';
```

Comment:
Is it worth adding this empty string as an insurance policy so we don't accidentally render the custom URI description text for DTC view in case the `DescriptionElement` accidentally gets added to the `uischema` despite the if check added below?
```   
if (challengeMethod !== CHALLENGE_METHOD.CHROME_DTC) {
    uischema.elements.push(descriptionElement);
    uischema.elements.push(openOktaVerifyButton);
}
```

Reply:
imo I think its fine, I think it also makes it more clear and consistent that the chrome dtc method should not have a description

Reply:
This isn't a bad idea, but you could also see if moving the `default` case to a case for `CHALLEGE_METHOD.CUSTOM_URI` and then change the fallthrough `default` to always return `''`

Reply:
That makes more sense, thanks for the suggestion

Reply:
I'm just sure if for some reason we depend on the `default` case to be the custom uri text. You could try it and see what happens in tests

Reply:
If you do this may also want to consider updating the other switch case above as well

Reply:
IIRC I dont think custom uri actually needs to be the default case so it should be fine